roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -97,3 +97,7 @@ cifmw_cephadm_grafana_admin_user: 'admin'
 cifmw_cephadm_grafana_admin_password: '/home/grafana_password.yml'
 cifmw_cephadm_prometheus_port: 9092
 cifmw_cephadm_grafana_port: 3100
+cifmw_cephadm_alertmanager_container_image: "quay.io/prometheus/alertmanager:v0.25.0"
+cifmw_cephadm_grafana_container_image: "quay.io/ceph/ceph-grafana:9.4.7"
```

Comment:
nit: you can use {{ cifmw_cephadm_container_ns }} here
and maybe you can define some other value for quay.io/prometheus

Reply:
done